docs/registry/create/_index.md

```diff
@@ -685,17 +690,62 @@ Make sure to set up a Python virtual environment in the directory your module is
 
     # Be sure to use `exec` so that termination signals reach the python process,
     # or handle forwarding termination signals manually
-    exec $PYTHON <your-src-dir-if-inside>/main.py $@
-   ```
+    exec $PYTHON -m src.main $@
+```
 
-   To make your shell script executable, run the following command in your terminal:
+To make your shell script executable, run the following command in your terminal:
 
-   ```sh { class="command-line" data-prompt="$"}
-   sudo chmod +x <your-file-path-to>/<run.sh>
-   ```
+```sh { class="command-line" data-prompt="$"}
+sudo chmod +x <your-file-path-to>/<run.sh>
+```
 
 See [Prepare your Python Virtual Environment](/program/python-venv/) for more information.
 
+{{% /tab %}}
+{{% tab name="nuitka" %}}
+
+Install [a C compiler on your machine](https://github.com/Nuitka/Nuitka#c-compiler).
```

Comment:
```suggestion
Install a [supported C compiler](https://github.com/Nuitka/Nuitka#c-compiler) on your machine.
```

Supernit: link just the nouns or terms themselves (with supporting adjectives if applicable), unless we are linking to a procedural or task, then fuller sentences and verbs are appropriate. Also adding `supported` because link mentions specific compiler requirements for use with `nutika`.